config/environments/development.rb

```diff
@@ -65,18 +65,15 @@
   # config.action_cable.disable_request_forgery_protection = true
 
   # Devise mailer.
-  config.action_mailer.default_url_options = {host: "localhost", port: 3000}
+  config.action_mailer.default_url_options = {host: "localhost", port: 3000, locale: nil}
 
   # Configure host for URL helpers.
-  Rails.application.routes.default_url_options = {host: "localhost", port: 3000}
+  Rails.application.routes.default_url_options = {host: "localhost", port: 3000, locale: nil}
```

Comment:
Do we need these for production, too?

Reply:
Yes, we need.
The current setting in `production.rb` is `Rails.application.routes.default_url_options[:host] = ENV["HOST"]`.
Can it be changed to the hash format like this?

Reply:
Yeah, go for it. We won't need to set `port`, so just `host` and `locale`.

Reply:
Done!